youtube_dl/extractor/tvnoe.py

```diff
@@ -0,0 +1,44 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .jwplatform import JWPlatformBaseIE
+from ..utils import clean_html, get_element_by_class, js_to_json
+
+
+class TVNoeIE(JWPlatformBaseIE):
+    _VALID_URL = r'https?://(www\.)?tvnoe\.cz/video/(?P<id>[0-9]+)'
+    _TEST = {
+        'url': 'http://www.tvnoe.cz/video/10362',
+        'md5': 'aee983f279aab96ec45ab6e2abb3c2ca',
+        'info_dict': {
+            'id': '10362',
+            'ext': 'mp4',
+            'series': 'Noční univerzita',
+            'title': 'prof. Tomáš Halík, Th.D. - ' +
+                     'Návrat náboženství a střet civilizací',
+            'description': 'md5:f337bae384e1a531a52c55ebc50fff41',
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        iframe_url = self._search_regex(r'<iframe[^>]+src="([^"]+)"',
+                                        webpage, 'iframe src attribute')
+
+        ifs_page = self._download_webpage(iframe_url, video_id)
+        jwplayer_data = self._parse_json(self._find_jwplayer_data(ifs_page),
+                                         video_id, transform_source=js_to_json)
+        info_dict = self._parse_jwplayer_data(
+            jwplayer_data, video_id, require_title=False, base_url=iframe_url)
+
+        info_dict.update({
+            u'id': video_id,
+            u'title': clean_html(
+                get_element_by_class('field-name-field-podnazev', webpage)),
+            u'description': clean_html(get_element_by_class('field-name-body',
+                                                            webpage)),
+            u'series': clean_html(get_element_by_class('title', webpage))
```

Comment:
```u``` prefix is not necessary as ```from __future__ import unicode_literals``` has the same effect, and such a syntax is not available in Python 3.2.